lib/danger/danger_core/plugins/dangerfile_bitbucket_server_plugin.rb

```diff
@@ -150,7 +150,9 @@ def head_commit
     # @return [String]
     #
     def html_link(paths, full_path: true)
-      create_link(paths, full_path) { |href, text| create_html_link(href, text) }
+      # TODO: Enable html formatting when BitBucketServer could handle it
+      #create_link(paths, full_path) { |href, text| create_html_link(href, text) }
+      markdown_link(paths, full_path: full_path)
```

Comment:
Tests are failing at this point. I have two options how to fix them:
* completely remove html formatting code with tests
* move this fix to `danger-xcode_summary` plugin repo and call a proper method based on chosen RequestSource